.gitignore

```diff
@@ -84,6 +84,7 @@ celerybeat-schedule
 
 # virtualenv
 .venv
+pyenv/
```

Comment:
Since this is custom rather than part of https://github.com/github/gitignore/blob/master/Python.gitignore, it should be at the top of the file